### PR TITLE
Fix null pointer exception in RFI request flow

### PR DESCRIPTION
**Pull request by jinnnn9**
ref: https://github.com/nanoPayinc/NANOPAY/pull/14107

this needs to be in v3.17 please @jlhughes

### CODE REVIEW

File: src/foam/nanos/ticket/Ticket.js [hunk 1]
```diff
@@ -452,6 +452,17 @@ foam.CLASS({
         }
       `
     },
+    {
+      name: 'getSystemX',
```
Comment (jlhughes) [inline src/foam/nanos/ticket/Ticket.js]: Why is this required? 
what is getX()

**Review by jlhughes — APPROVED** [review]

**Review by nanoMichal — APPROVED** [review]
